docs/user_guide/assessment.rst

```diff
@@ -394,15 +394,45 @@ For more examples, please
 see the :ref:`sphx_glr_auto_examples_plot_new_metrics.py` notebook in the
 :ref:`examples`.
 
+.. _plot:
+
+Plotting grouped metrics
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The simplest way to visualize grouped metrics from the :class:`MetricFrame` is
+to take advantage of the inherent plotting capabilities of
+:class:`pandas.DataFrame`:
+
+
+.. Below the code-block contains the code shown on the website.
+   It should be kept in sync with the plot_quickstart.py example.
+
+.. code-block:: python
+
+    metrics = {
+        'accuracy': accuracy_score,
+        'precision': precision_score,
+        'recall': recall_score,
+        'false positive rate': false_positive_rate,
+        'true positive rate': true_positive_rate,
+        'selection rate': selection_rate,
+        'count': lambda y_true, y_pred: y_true.shape[0]}
+    metric_frame = MetricFrame(metrics, y_true, y_pred, sensitive_features=sex)
+    metric_frame.by_group.plot.bar(
+        subplots=True, layout=[3,3], legend=False, figsize=[12,8],
+        title='Show all metrics')
+
+.. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
+    :target: auto_examples/plot_quickstart.html
+    :align: center 
+
 
 .. _dashboard:
 
 Fairlearn dashboard
 -------------------
 
-.. currentmodule:: fairlearn.widget
-
-The Fairlearn dashboard is a Jupyter notebook widget for assessing how a
+The Fairlearn dashboard was a Jupyter notebook widget for assessing how a
```

Comment:
Should we just remove this from here? This PR could fix the docs, and the other PR could remove the implementation.

Reply:
You mean remove the entire section on the widget, or remove the changes to this section (so that they're made in the widget removal PR)?
I think you mean the former. I think it wouldn't hurt to keep this short section with just a pointer to the new location in case someone is looking for it.